packages/frontend/src/app/repository/repository.component.spec.ts

```diff
@@ -177,4 +185,68 @@ describe('RepositoryComponent', () => {
 
     expect(location.path()).toEqual(RouteProvider.routes._404.link());
   }));
+
+  it('should hide the spinner when tests list is ready', fakeAsync(() => {
```

Comment:
I've said this before, but truly appreciate how thorough we're all being with tests. This will create a great foundation for future folks contributing to the project.